modular_citadel/code/modules/clothing/head/head.dm

```diff
@@ -17,4 +17,13 @@
 //The "pocket" for the M1 helmet so you can tuck things into the elastic band
 
 /datum/component/storage/concrete/pockets/tiny/spacenam
-	attack_hand_interact = TRUE		//So you can actually see what you stuff in there
\ No newline at end of file
+	attack_hand_interact = TRUE		//So you can actually see what you stuff in there
+
+/obj/item/clothing/head/assu_helmet
+	name = "ASSU helmet"
+	icon = 'modular_citadel/icons/obj/clothing/assu.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/mob/clothing/assu.dmi'
```

Comment:
Just move this item to `code/modules/clothing/head/misc.dm`, remove these two lines and add the sprites to the standard `clothing/hats.dmi` and `mob/head.dmi` instead.
It's annoying to keep track of the dozens already existing snowflake .dm and .dmi.